components/usage/pkg/controller/reconciler_test.go

```diff
@@ -167,19 +168,30 @@ func TestUsageReportConversionToDBUsageRecords(t *testing.T) {
 				teamAttributionID: {
 					db.WorkspaceInstanceForUsage{
 						ID:                 instanceId,
-						UsageAttributionID: teamAttributionID,
+						WorkspaceID:        workspaceID,
+						OwnerID:            ownerID,
+						ProjectID:          sql.NullString{},
 						WorkspaceClass:     defaultWorkspaceClass,
+						Type:               db.WorkspaceType_Prebuild,
+						UsageAttributionID: teamAttributionID,
 						CreationTime:       creationTime,
 						StoppedTime:        stoppedTime,
 					},
 				},
 			},
 			Expected: []db.WorkspaceInstanceUsage{{
-				InstanceID:    instanceId,
-				AttributionID: teamAttributionID,
-				StartedAt:     creationTime.Time(),
-				StoppedAt:     sql.NullTime{Time: stoppedTime.Time(), Valid: true},
-				CreditsUsed:   470,
+				InstanceID:     instanceId,
+				AttributionID:  teamAttributionID,
+				UserID:         ownerID,
```

Comment:
What would be the ownerID here for a prebuild and do we expect there to be one?

Reply:
```
SELECT wsi.id, ws.type, ws.ownerId from d_b_workspace as ws, d_b_workspace_instance as wsi where wsi.workspaceId = ws.id AND ws.type = 'prebuild' LIMIT 100;
```
https://console.cloud.google.com/bigquery?sq=441577903635:724ea3fda1724aab80bae2f7babc5758

Prebuilds have owners. Sadly they are the user who setup the prebuild but oh well.

Reply:
> the user who setup the prebuild

I guess for the billing session this information will be irrelevant, but fine to keep here in case for future purposes.